src/soft_start.cpp

```diff
@@ -17,16 +17,15 @@ float SoftStart::Constrain(float values, float min_val, float max_val) {
   return fmin(fmax(values, min_val), max_val);
 }
 
-void SoftStart::ConstrainTorques(std::vector<float> &torques, int count) {
-  if (count > duration_) {
+void SoftStart::ConstrainTorques(std::vector<float> &torques, float time_since_start_ms) {
+  if ((time_since_start_ms) > duration_ms_) { // ramp_timer returns in us
```

Comment:
comment does not make sense

Reply:
Fixed in 21511f7a0d652fe35a9fc578d4b151ac0b83c9d4